### PR TITLE
Page/search

### PR DESCRIPTION
**Pull request by ehn77**

### CODE REVIEW

File: src/app/api/services/searchService.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import pool from "../../connection.js";
+
+const getTournamentsByName = async (name) => {
+  try {
+    const result = await pool.query(
+      `SELECT * FROM SELECT * FROM tournaments WHERE name ILIKE $1`,
```
Comment (lm-rice) [inline src/app/api/services/searchService.js]: huh? why are we selecting * from a select *

File: src/app/api/controllers/searchController.js
Comment (diep-steven) [inline src/app/api/controllers/searchController.js:1]: Make sure to change thee name of participantsService to searchService

File: src/app/api/controllers/searchController.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import express from "express";
+<<<<<<< HEAD
+import participantsService from '../services/searchService.js';
+=======
+import searchService from "../services/searchService.js";
+>>>>>>> 34722df (syntax error fix)
```
Comment (lm-rice) [inline src/app/api/controllers/searchController.js]: Missed this conflict

File: src/app/api/services/userService.js [hunk 3]
```diff
@@ -16,6 +16,15 @@ const createUser = async (username, email, password) => {
...
     );
+
+  try {
+    const result = await pool.query(
+      `INSERT INTO users(username, email, password_hash)
+             VALUES ($1, $2, crypt($3, gen_salt('md5')))
+             RETURNING *`,
+      [username, email, password],
+    );
+    return result.rows[0];
```
Comment (lm-rice) [inline src/app/api/services/userService.js]: this should not have been added back, assume something went wrong with the rebase

File: src/app/api/services/userService.js [hunk 4]
```diff
@@ -77,8 +88,13 @@ const deleteUser = async (id) => {
...
   getUserById,
   updateUser,
   deleteUser,
-};
\ No newline at end of file
+};
+  getUserById,
+  updateUser,
+  deleteUser,
+};
```
Comment (lm-rice) [inline src/app/api/services/userService.js]: this is not valid syntax, assuming another rebase problem

**Review by lm-rice — APPROVED** [review]

**Pull request merged**